dist/index.js

```diff
@@ -496,6 +542,76 @@ class IssuesProcessor {
+                        const lastCommentCreatedAt = lastComment.created_at || IssuesProcessor._getNow();
+                        const daysSinceCreated = IssuesProcessor._getDaysSince(lastCommentCreatedAt);
+                        issueLogger.info(`Days since the last comment was posted: ${daysSinceCreated.toFixed(2)}`);
+                        if (!isBotComment &&
+                            !IssuesProcessor._updatedSince(lastCommentCreatedAt, daysBeforeReplyReminder)) {
+                            const lastCommentPostedByMaintainer = lastComment.user
+                                ? this.isPostedByMaintainer(lastComment.user.login)
+                                : false;
+                            issueLogger.info(`Did a maintainer post the last comment? ${lastCommentPostedByMaintainer}`);
+                            if (lastCommentPostedByMaintainer) {
+                                const mention = issue.user ? `@${issue.user.login}` : '';
+                                yield this.createComment(issue, `${mention} Any updates here?`);
+                                return;
+                            }
+                            else {
+                                yield this.createComment(issue, `${reminderToMaintainers}. Please reply to comments.`);
+                                return;
+                            }
+                        }
+                        // We should not stale an issue if it has no comments from maintainers
+                        const hasMaintainerComment = comments.some(({ user }) => user ? this.isMaintainer(user.login) : false);
+                        if (!hasMaintainerComment) {
+                            issueLogger.info('This issue has no comments from maintainers');
+                            return;
+                        }
+                        if (isBotComment &&
+                            ((_d = lastComment.body) === null || _d === void 0 ? void 0 : _d.includes(reminderToMaintainers))) {
+                            issueLogger.info('The last comment is a reminder to maintainers posted by a bot');
+                            return;
+                        }
+                    }
+                    else {
+                        const daysSinceCreated = IssuesProcessor._getDaysSince(issue.created_at);
+                        issueLogger.info(`Days since this issue was created: ${daysSinceCreated.toFixed(2)}`);
+                        const hasMaintainerAssignee = issue.assignees.some(user => this.isMaintainer(user.login));
+                        if (!hasMaintainerAssignee &&
+                            !IssuesProcessor._updatedSince(issue.created_at, daysBeforeAssigneeReminder)) {
+                            issueLogger.info('This issue has no assignees');
+                            yield this.createComment(issue, `${reminderToMaintainers}. Please assign a maintainer to this issue and start triaging.`);
```

Comment:
Can we tag the core maintainers (@dbczumar, @harupy , @WeichenXu123 , @BenWilson2) in these comments?